.github/PULL_REQUEST_TEMPLATE/release_template.md

```diff
@@ -0,0 +1,32 @@
+### Manual Release Steps
+
+## Features ⭐
+
+
+
+## Visualisations 📊 
```

Comment:
Have we used this section in recent history? Can probably drop it, as visualisations have become Maui responsibility and configured via the front end, rather than dev work.

Reply:
I only had a look through the most recent six releases (ignoring ones that were closed without merging), and it was used in 2 of those 6. Do you reckon remove it? I’ll defer to your call here

Reply:
Can you link me to those where it was used?

Reply:
These are from the last 10 releases:

- #4921
- #4697
- #4360

Reply:
Yeah I suspected it'd be like that - those aren't visualisations, they're "features" and "tweaks" to the types of visualisations we can display. Historically, an entry would look like "Malaria cases per month", etc. The section can definitely go.